src/types/permissions.ts

```diff
@@ -0,0 +1,12 @@
+export type PermissibleQueryResponse = {
+  mutationType: {
+    fields: PermissibleQuery[];
+  };
+  queryType: {
```

Comment:
What's the difference between `mutationType` and `queryType`? Know it's something coming from the backend but just curious. Is `queryType` a list of read access fields and `mutationType` the list of full CRUD access fields for the user?

Reply:
Yeah my understanding is that the `mutationType` queries are the CUD and `queryType` are the R fields. Combined they make up the full CRUD list. However, the `simulate` query I think is the only `queryType` that triggers things to mutate, but I don't think actually directly mutates anything.

Reply:
Maybe worth dropping a comment in there about it?

Reply:
A comment directly next to the gql function or on the effects function invoking it?